### PR TITLE
Update gemini .repl and .resc files

### PR DESCRIPTION
**Pull request by jorge-rs**

### CODE REVIEW

File: platforms/cpus/gemini.repl [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//Gemini SoC platform
+
+//Memory
+ddr: Memory.MappedMemory @ sysbus 0x0
+    size: 0x80000000
+
+sram: Memory.MappedMemory @ sysbus 0x80000000
+    size: 0x80000
```
Comment (rahul-r-shah) [inline platforms/cpus/gemini.repl]: Isnt size 256KB divided in 4 banks of 64 KB each starting from 0x80000000, 0x80010000, 0x80020000, 0x80030000?

**Review by rahul-r-shah — CHANGES_REQUESTED** [review]
1 change that I see.

Besides need to make sure that the correct infrastructure commit from main is being referred to

File: platforms/cpus/gemini.repl (hunk 1 above)
Comment (jorge-rs) [reply platforms/cpus/gemini.repl]: Thanks for the observation. I overlooked this value. Changed accordingly.

**Comment by jorge-rs** [comment]
> 1 change that I see.
> 
> Besides need to make sure that the correct infrastructure commit from main is being referred to

Right. I will update the renode-infrastructure here as soon as we have the other PR done

**Comment by rahul-r-shah** [comment]
> > 1 change that I see.
> > Besides need to make sure that the correct infrastructure commit from main is being referred to
> 
> Right. I will update the renode-infrastructure here as soon as we have the other PR done

I guess we dont need to do anything. We just need to make sure that the infrastructure submodule points to the main branch of the RS infrastructure repository. Also we need to ensure that all the infrastructure related changes are merged into the RS Infrastructure repository on time or before we merge the changes to the renode_rs if the renode_rs is dependent on the Infrastructure changes

**Review by rahul-r-shah — APPROVED** [review]
LGTM

**Pull request merged**